groups/managers.py

```diff
@@ -83,3 +135,22 @@ class CommentManager(PolymorphicManager, CommentManagerMixin):
     """PolymorphicManager for BaseComments with custom methods."""
     def get_queryset(self):
         return CommentQuerySet(self.model, using=self._db)
+
+
+class WithinDaysUserQuerySetMixin:
+    """
+    A mixin that adds methods for returning users that have recently posted.
+
+    Can be mixed into a Manager or a QuerySet.
+    """
+    def within_days(self, days=DEFAULT_WITHIN_DAYS):
+        """All users that created a comment within the last `days` days."""
+        return self.since(get_threshold_date(days))
+
+    def within_time(self, timedelta):
+        """All users that created a comment within the last `timedelta`."""
+        return self.since(get_threshold_delta(timedelta))
+
+    def since(self, when):
+        """All the comments in this queryset posted since `when`."""
+        return self.filter(comments__date_created__gte=when).distinct()
```

Comment:
You mean 'similar methods'?  I'm not sure what you're getting at.  Or do you mean reusing this mixin itself?